packages/realm/src/assert.ts

```diff
@@ -71,14 +71,14 @@ assert.boolean = (value: unknown, target?: string): asserts value is boolean =>
   }
 };
 
-assert.bigInt = (value: unknown, target?: string): asserts value is bigint => {
-  if (typeof value !== "bigint") {
+assert.bigInt = (value: unknown, target?: string): asserts value is binding.Int64 => {
+  if (binding.Int64.isInt(value)) {
     throw new TypeAssertionError("a bigint", value, target);
   }
 };
 
-assert.numberOrBigInt = (value: unknown, target?: string): asserts value is number => {
-  if (typeof value !== "number" && typeof value !== "bigint") {
+assert.numberOrBigInt = (value: unknown, target?: string): asserts value is number | binding.Int64 => {
+  if (typeof value !== "number" && binding.Int64.isInt(value)) {
```

Comment:
```suggestion
  if (typeof value !== "number" && !binding.Int64.isInt(value)) {
```

Reply:
D'oh! Luckily, it looks like neither `assert.bigInt` nor `assert.numberOrBigInt` are actually used anywhere...

Reply:
I double checked all usages of `isInt`. Those were the only ones that were using `!==` before. The rest were correctly converted.

Reply:
Alright 👍  Yeah they probably weren't used 😛